src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -9,9 +9,10 @@
     public static final String MESSAGE_SUCCESS = "Sorted all pets!";
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Sorts the pet list by the column specified.\n"
-            + "Columns that can be sorted: OwnerName (/o), Name (/n) \n"
-            + "Parameters: FIRST LETTER OF COLUMN NAME \n"
-            + "Example: " + COMMAND_WORD + " /o";
+            + "Columns that can be sorted by the following 5 fields:\n"
+            + "1.owner\n 2.name\n 3.app\n 4.pick up\n 5.drop off\n"
```

Comment:
Maybe add a space after each number point, and capitalize the fields for better readability?

Reply:
Noted. Will add a spacing but will not be capitalising as the fields are case sensitivity. Will make changes to message for better clarity.